src/Theme/OpenEuropaBoostrapThemePreRender.php

```diff
@@ -1,16 +1,16 @@
 <?php
```

Comment:
Same here for the filename, class and usages:
- OpenEuropaBoostrapThemePreRender.php > OpenEuropaBootstrapThemePreRender.php
- class OpenEuropaBoostrapThemePreRender > class OpenEuropaBootstrapThemePreRender
- and calls to that class